### ISSUE TITLE
cannot find BaseSystem.dmg

### ISSUE THREAD

**Issue by a2drew4u** [foxlet/macOS-Simple-KVM#257]
i just tried to download big sur and it has downloaded BaseSystem.dmg but it has not downloaded to the macOS-Simple-KVM-master folder or the base system folder yet when jumpstart.sh was running i saw the free space decrease and thinking it was a glitch i deleted the macOS-Simple-KVM-master folder and emptied the recycle bin and rebooted my laptop and the free space has not gone up, is there a way you could help me find where it downloaded to

**Comment by a2drew4u** [comment]
the reason i want to find the location is because this has happened to me in the past before with jumpstart.sh for catilina and high sierra and now it is up to about 30 gb of unuseable space the reason i am saying this the third time is because i have tried averything i could think of and still cant find how to get back the 30 gb

**Comment by mtedaldi** [comment]
It seems to not download the BaseSystem.dmg but an 4GiB InstallAssistant.pkg file into ./tools/FetchMacOS/BaseSystem/

That's how far I've made it. Did not manage to get the install package to work from within cataltina (the installer starts but does not let me select a drive to install to).

About the fre space issue: what was the exact command you entered (with path and all that) to get the package?

**Comment by a2drew4u** [comment]
yeah about that i reset my laptop because it was having some other issues also and now all of those problems are gone but what i did to do the command was open the file manager to macOS-Simple-KVM-master folder and clicked open in terminal then typed ./jumpstart.sh -b .
